sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -773,6 +768,11 @@ def get_blob_client(
                 :dedent: 12
                 :caption: Getting the blob client to interact with a specific blob.
         """
+        if isinstance(blob, BlobProperties):
+            warnings.warn(
+                "The use of a 'BlobProperties' instance is deprecated. Please use str instead.",
```

Comment:
Can we add a mention of the param `blob` here (and everywhere else). 

"The use of a 'BlobProperties' instance for param blob is deprecated. Please use str instead."